src/Validation/Casts/Value.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaraStrict\Validation\Casts;
```

Comment:
Není to tedy spíš Helpers?

Reply:
A dal bych to do Core\Helpers

At Database modul není závislý na validation

Reply:
Takže do `LaraStrict\Validation\Helpers`?